paper/paper.md

```diff
@@ -0,0 +1,149 @@
+---
+title: 'FlowerMD: Flexible Library of Organic Workflows and Extensible Recipes for Molecular Dynamics'
+tags:
+  - molecular simulation
+  - materials science
+  - molecular dynamics
+  - polymers
+  - HOOMD-blue
+authors:
+  - name: Chris Jones
+    orcid: 0000-0002-6196-5274
+    equal-contrib: true
+    affiliation: 1
+  - name: Marjan Albooyeh
+    orcid: 0009-0001-9565-3076
+    equal-contrib: true
+    affiliation: 1
+  - name: Rainier Barrett
+    orcid: 0000-0002-5728-9074
+    equal-contrib: false
+    affiliation: 1
+  - name: Eric Jankowski
+    orcid: 0000-0002-3267-1410
+    corresponding: true
+    affiliation: 1
+affiliations:
+ - name: Boise State University, Boise, ID, USA
+   index: 1
+date: 01 January 2001
+bibliography: paper.bib
+---
+# Summary
+`flowerMD` is a package for reproducibly performing multi-stage HOOMD-blue [@hoomd_2019]
+simulation workflows. It enables the programmatic specification of tasks including
+definition of molecular structures, forcefield definition and application and chaining
+together simulation stages (e.g., shrinking, equilibration, simulating a sequence
+of ensembles, tensile testing, etc.) through an extensible set of Python classes.
+The modular design supports a library of workflows for organic
+macrmolecular and polymer simulations. Tutorials are provided to demonstrate
+package features and flexibility.
+
+
+# Statement of need
+
+High-level programmatic specifications of molecular simulation workflows are
+needed for two reasons. First, they provide the information necessary for a
+simulation study to be reproducible, and second, they minimize the cognitive
+load of getting started with running simulations.
+For a researcher new to molecular simulations, building the necessary set
+of computational tools needed to actually perform experiments simultaneously:
+(a) requires skills and knowledge different from those needed to do research, and
+(b) involves repeating work that others have already done.
+
+This is a well recognized problem, and recent advances in well-documented
+open-source tools have made the programmatic specification of
+molecular simulation components easier than ever
+[@hoomd_2019; @lammps_2022; @gromacs_2015; @mbuild_2016; @gmso;
```

Comment:
Do we need a couple more citations here? Openmm, parmed, pyprism, pysoftk?

Reply:
Pysoftk is already cited. I will add the other ones.